spec/danger_packwerk/danger_packwerk_spec.rb

```diff
@@ -46,6 +51,13 @@ module DangerPackwerk
             location: Packwerk::Node::Location.new(12, 5)
           )
         end
+        let(:depended_on_reference) do
+          sorbet_double(
```

Comment:
I had to run this down. I didn't know `sorbet_double` was a thing. It's actually just defined the spec_helper

Reply:
oh, I didn't even notice this was different. cool!